controllers/drplacementcontrol.go

```diff
@@ -499,17 +502,18 @@ func (d *DRPCInstance) readyToSwitchOver(homeCluster string) bool {
 		return !ready
 	}
 
-	clusterDataReadyCondition := findCondition(vrg.Status.Conditions, VRGConditionTypeClusterDataReady)
-	if clusterDataReadyCondition == nil {
+	clusterDataProtectedCondition := findCondition(vrg.Status.Conditions, VRGConditionTypeClusterDataProtected)
+	if clusterDataProtectedCondition == nil {
 		d.log.Info("VRG ClusterData condition not available", "cluster", homeCluster)
 
 		return !ready
 	}
 
+	// Allow switch over when PV data is ready and the cluster data is protected
```

Comment:
We should check for both conditions `ClusterDataProtected` and `ClusterDataReady`. Currently `ClusterDataReady` is marked if all VRs for PVCs report back as ready (available). We still backup a PV for a PVC if it's condition is not `ClusterDataReady` and hence mark `ClusterDataProtected` as true if all PVs were protected.

So for relocation both conditions should be true, to ensure VRs have flipped state as required and we have protected the PV cluster data.

Ideally in the future, VR should report if it established resync for a Primary and so we can have a `DataProtected` condition to check, instead of the current `DataReady` condition.

Reply:
Hmmm, `ClusterDataReady` is only set when PV is restored successfully.  Why would we check for it if we are relocating? I think you mean `DataReady` which we do check for.  Confusing terms...

Reply:
IOW, for relocation, we have to make sure that `DataReady` is true and `ClusterDataProtected` is true.  This is for the source cluster.  For the target cluster, at some point, we would check if `ClusterDataReady` is true.

Reply:
> Hmmm, `ClusterDataReady` is only set when PV is restored successfully. Why would we check for it if we are relocating? I think you mean `DataReady` which we do check for. Confusing terms...

True... I mean `DataReady` in the initial comment.